core/assets/bundles/bundle_fr.properties

```
@@ -1832,19 +1832,19 @@ hint.depositItems = Pour déposer des ressources dans le Noyau, déplacez-les de
 hint.respawn = Pour réapparaître en tant que vaisseau, pressez [accent][[V][].
 hint.respawn.mobile = Vous avez pris le contrôle d'une unité/structure. Pour réapparaître en tant qu'unité de base, [accent]touchez l'avatar en haut à gauche.[]
 hint.desktopPause = Pressez [accent][[Espace][] pour mettre en pause et reprendre le jeu.
-hint.breaking = Maintenez [accent]Clic-droit[] pour détruire des blocs.
+hint.breaking = Maintenez [accent][[Clic-droit][] pour détruire des blocs.
 hint.breaking.mobile = Activez le \ue817 [accent]marteau[] en bas à droite, Touchez pour détruire des blocs.\n\nRetenez votre doigt pendant une seconde et déplacez-le pour détruire les blocs dans la zone de sélection.
 hint.blockInfo = Pour afficher les informations relatives à un bloc, il suffit de le sélectionner dans le [accent]menu de construction[], puis de cliquer sur le bouton [accent][[?][] à droite.
 hint.derelict = [accent]Les structures abandonnées[] sont des vestiges brisés d'anciennes bases qui ne fonctionnent plus. Ces structures peuvent être [accent]déconstruites pour obtenir des ressources.
 hint.research = Utilisez le bouton \ue875 [accent]Recherche[] pour rechercher de nouvelles technologies.
 hint.research.mobile = Utilisez le bouton \ue875 [accent]Recherche[] dans le \ue88c [accent]Menu[] pour rechercher de nouvelles technologies.
 hint.unitControl = Retenez [accent][[Ctrl-gauche][] et [accent]cliquez[] pour contrôler une tourelle ou une unité alliée.
-hint.unitControl.mobile = [accent][[Tapez][] 2 fois une tourelle ou une unité alliée pour la contrôler.
-hint.unitSelectControl = Pour contrôler les unités, entrez en mode [accent]« Commande »[] en pressant [accent]Maj gauche.[]\nEn mode « Commande », cliquez et faites glisser la souris pour sélectionner des unités. Faites un [accent]Clic droit[] à un emplacement ou une cible pour que les unités s'y déplacent.
+hint.unitControl.mobile = [accent]Tapez[] 2 fois une tourelle ou une unité alliée pour la contrôler.
+hint.unitSelectControl = Pour contrôler les unités, entrez en mode [accent]« Commande »[] en pressant [accent][[Maj gauche][].\nEn mode « Commande », cliquez et faites glisser la souris pour sélectionner des unités. Faites un [accent]Clic droit[] à un emplacement ou une cible pour que les unités s'y déplacent.
```

```suggestion
hint.unitSelectControl = Pour contrôler les unités, entrez en mode [accent]« Commande »[] en pressant [accent]Maj gauche[].\nEn mode « Commande », cliquez et faites glisser la souris pour sélectionner des unités. Faites un [accent]Clic droit[] à un emplacement ou une cible pour que les unités s'y déplacent.
```
Encore.
https://github.com/phinner/mindustry/blob/e4253bcbf8fdc638cc52247ac553b5ed697447d4/core/assets/bundles/bundle.properties#L1843